test/E2ETest/Microsoft.Test.E2E.AspNet.OData/BulkOperation/BulkOperationController.cs

```diff
@@ -249,15 +249,16 @@ public ITestActionResult PatchEmployees([FromBody] DeltaSet<Employee> coll)
 
         [ODataRoute("Employees")]
         [HttpPost]
+        [EnableQuery]
         public ITestActionResult Post([FromBody] Employee employee)
         {
-            InitEmployees();
+            //InitEmployees();
 
-            var handler = new EmployeeAPIHandler();
+            //var handler = new EmployeeAPIHandler();
 
-            handler.UpdateLinkedObjects(employee, Request.GetModel());
+            //handler.UpdateLinkedObjects(employee, Request.GetModel());
 
-            return Ok(employee);
+            return Created(employee);
```

Comment:
Maybe you should have created a new test to validate this new fix instead of modifying an existing case since the existing behaviour should continue to be supported.

Reply:
The original tests are not being used since they are part of Deep Insert work which is in progress. The final tests should be as modified